packages/govuk-frontend/src/govuk/components/task-list/_index.scss

```diff
@@ -8,6 +8,7 @@
     margin-top: 0;
     @include govuk-responsive-margin(6, "bottom");
     padding: 0;
+    overflow: scroll;
```

Comment:
Isn't this the rule that puts always-visible scrollbars on Windows? 😬

Both horizontal and vertical

Reply:
I think `<select>` was the best example in:

* https://github.com/alphagov/govuk-frontend/issues/1009

Users could add long `<option>` items with long widths, breaking the layout

Ideally we try our best to never add scroll bars?

Reply:
You can configure macOS to show scroll bars too

<img width="482" alt="Scroll bars on macOS" src="https://github.com/alphagov/govuk-frontend/assets/415517/b3f3b3ad-7ee4-43c4-a4f6-aa3849c13f3c">

<br>
<br>

At the moment both horizontal + vertical scroll bars show up?

<img width="222" alt="Example showing scroll bars" src="https://github.com/alphagov/govuk-frontend/assets/415517/7608e075-ade3-4f46-a070-e258db4d803b">

Could narrow it down to `overflow-x` but `max-width` + text wrapping would be better

Shall we drop the commit for another PR?

Reply:
I think so. I'll drop the commit, merge this and make an issue.